sbt/src/main/scala/stryker4s/sbt/Stryker4sSbtRunner.scala

```diff
@@ -123,6 +123,7 @@ class Stryker4sSbtRunner(
         val javaOpts = extractTaskValue(Test / javaOptions, "javaOptions")
 
         val frameworks = extractTaskValue(Test / loadedTestFrameworks, "test frameworks").values.toSeq
+        if (frameworks.isEmpty) log.warn("No test frameworks found via loadedTestFrameworks")
```

Comment:
I'd like this to be a bit more informative about what it means for the user. Most people probably don't know what `loadedTestFrameworks` is or what impact it has on Stryker4s. Maybe we can add a message that this likely means no tests will run and result in a bogus score? Ideally also some way to fix it, but I don't know what that could be (maybe configuring `testFrameworks` manually?)

Reply:
Searched for a while and found the change from sbt-crossproject to projectmatrix, don't know what causes that situation or how to diagnose it further at the moment. UnU

Will change the message as suggested.